gettsim/transfers/benefit_checks/vermoegens_checks.py

```diff
@@ -179,3 +185,36 @@ def arbeitsl_geld_2_vermög_freib_hh(
         + haushaltsgröße_hh * arbeitsl_geld_2_params["vermögensfreibetrag_austattung"]
     )
     return out
+
+
+def arbeitsl_geld_2_vermög_freib_hh_ab_2023(
+    arbeitsl_geld_2_params: dict,
+    haushaltsgröße_hh: int,
+    bürgerg_bezug_vorj: float,
+) -> float:
+    """Calculate actual wealth exemptions since 2023.
+
+    Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
+
+    Parameters
+    ----------
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+    haushaltsgröße_hh
+        See :func:`haushaltsgröße_hh`.
+    bürgerg_bezug_vorj
+        See basic input variable :ref:`bürgerg_bezug_vorj <bürgerg_bezug_vorj>`.
+
+
+    Returns
+    -------
+
+    """
+    params = arbeitsl_geld_2_params["schonvermögen_bürgergeld"]
+    if bürgerg_bezug_vorj == 0:
```

Comment:
Descripe here, or even better in docstring of function Karenzzeit (something like: "during the first year (Karenzzeit), the wealth exemption is substantially larger")